server/server.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const models = require('../database/models.js');
+
+const app = express();
+var port = 3000;
+
+app.use(cors());
+app.use(express.static('../public'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+
+app.get('/bookedTimes', (req, res) => {
```

Comment:
Consider perhaps using route parameters ([link](https://expressjs.com/en/guide/routing.html)) instead of query parameters 
`app.get('/users/:userId/books/:bookId', function (req, res) {
  res.send(req.params)
})`

Reply:
I made the appropriate changes to the API endpoints. Thanks for the tips.